vterm.el

```diff
@@ -176,16 +177,26 @@ for different shell. "
   (interactive)
   (vterm-send-key "c" nil nil t))
 
+(defun vterm-undo()
```

Comment:
Space between function name and parameter list. I thought it wouldn't even work without a space in Elisp ?